processor/src/main/java/com/beust/version/VersionProcessor.java

```diff
@@ -79,8 +79,10 @@ private String findVersionValue(Version version) throws IOException {
             if (f.exists()) {
                 log("Found " + f);
                 Properties p = new Properties();
-                p.load(new FileReader(f));
+                FileReader r = new FileReader(f);
```

Comment:
Fair enough, so how about closing the descriptor in a `finally`, just to be safe?